Dfe.PrepareTransfers.Web.Integration.Tests/Dfe.PrepareTransfers.Web.Integration.Tests.csproj

```diff
@@ -4,6 +4,8 @@
         <TargetFramework>net6.0</TargetFramework>
 
         <IsPackable>false</IsPackable>
+
+        <UserSecretsId>0daec1ca-536f-4ee8-a353-1f995aa8da76</UserSecretsId>
```

Comment:
Does this want to be an env var or is it ok here?

Reply:
Its ok, visual studio adds these if the user secrets manager is used. There are no secrets for the project anyway, but actual secret values wouldn't be included.